docs/get-started/introduction.md

```diff
@@ -24,6 +24,8 @@ NFD consists of three software components:
 1. nfd-master
 1. nfd-worker
 1. nfd-topology-updater
+1. nfd-topology-gc
```

Comment:
Could you also adjust the `NFD consists of three software components:` text above

Also, add a new sub-section for "NFD-Topology-GC". Just a one sentence description is enough for now